releasenotes/notes/add_DD_RUNTIME_METRICS_RUNTIME_ID_ENABLED-8e9d518dfd0028f2.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
```

Comment:
Do we have to delete the old release note?

Reply:
I just wanted to delete it because the name of the file is incorrect now.